### PR TITLE
Minor edit: headings in the <label> page.

### PR DESCRIPTION
**Pull request by estelle**
* Labels are not only for input.
* added name to example
* added "implicit" and "explicit"

### CODE REVIEW

File: files/en-us/web/html/element/label/index.md [hunk 1]
```diff
@@ -17,12 +17,12 @@ The **`<label>`** [HTML](/en-US/docs/Web/HTML) element represents a caption for
 
 {{EmbedInteractiveExample("pages/tabbed/label.html", "tabbed-shorter")}}
 
-Associating a `<label>` with an {{htmlelement("input")}} element offers some major advantages:
+Associating a `<label>` with an {{htmlelement("input")}} element (or other control) offers some major advantages:
```
Comment (dipikabh) [inline files/en-us/web/html/element/label/index.md:20]:
```suggestion
Associating a `<label>` element with an {{htmlelement("input")}} element (or other controls such as ) offers some major advantages, including the following:
```
could we give an example of other controls here?

File: files/en-us/web/html/element/label/index.md [hunk 2]
```diff
@@ -17,12 +17,12 @@ The **`<label>`** [HTML](/en-US/docs/Web/HTML) element represents a caption for
...
 
 - The label text is not only visually associated with its corresponding text input; it is programmatically associated with it too. This means that, for example, a screen reader will read out the label when the user is focused on the form input, making it easier for an assistive technology user to understand what data should be entered.
 - When a user clicks or touches/taps a label, the browser passes the focus to its associated input (the resulting event is also raised for the input). That increased hit area for focusing the input provides an advantage to anyone trying to activate it — including those using a touch-screen device.
 
-To associate the `<label>` with an `<input>` element, you need to give the `<input>` an `id` attribute. The `<label>` then needs a `for` attribute whose value is the same as the input's `id`.
+To explicitly associate the `<label>` with an `<input>` element, you need to give the `<input>` an `id` attribute. The `<label>` then needs a `for` attribute whose value is the same as the input's `id`.
```
Comment (dipikabh) [inline files/en-us/web/html/element/label/index.md:25]:
```suggestion
To explicitly associate a `<label>` element with an `<input>` element, you first need to add the `id` attribute to the `<input>` element. Next, you add the `for` attribute to the `<label>` element, where the value of `for' is the same as the `id` in the `<input>` element.
```

File: files/en-us/web/html/element/label/index.md [hunk 3]
```diff
@@ -65,15 +65,15 @@ There are no special styling considerations for `<label>` elements — structura
 
 ## Examples
 
-### Simple label example
+### Implicit label example
```
Comment (dipikabh) [inline files/en-us/web/html/element/label/index.md:68]:
```suggestion
### Defining an implicit label
```
...to use action-oriented example headings

File: files/en-us/web/html/element/label/index.md [hunk 4]
```diff
@@ -65,15 +65,15 @@ There are no special styling considerations for `<label>` elements — structura
...
 
 ```html
 <label>Click me <input type="text" /></label>
 ```
 
 {{EmbedLiveSample('Simple_label_example', '200', '50', '')}}
 
-### Using the "for" attribute
+### Explicit label: using the "for" attribute
```
Comment (dipikabh) [inline files/en-us/web/html/element/label/index.md:76]:
```suggestion
### Defining an explicit label with the "for" attribute
```

**Review by dipikabh — DISMISSED** [review]

File: files/en-us/web/html/element/label/index.md (hunk 1 above)
Comment (estelle) [inline files/en-us/web/html/element/label/index.md:20]:
```suggestion
Associating a `<label>` with a form control, such as {{htmlelement("input")}} or  {{htmlelement("textarea")}}   offers some major advantages:
```

File: files/en-us/web/html/element/label/index.md [hunk 5]
```diff
@@ -17,12 +17,12 @@ The **`<label>`** [HTML](/en-US/docs/Web/HTML) element represents a caption for
...
 
 - The label text is not only visually associated with its corresponding text input; it is programmatically associated with it too. This means that, for example, a screen reader will read out the label when the user is focused on the form input, making it easier for an assistive technology user to understand what data should be entered.
 - When a user clicks or touches/taps a label, the browser passes the focus to its associated input (the resulting event is also raised for the input). That increased hit area for focusing the input provides an advantage to anyone trying to activate it — including those using a touch-screen device.
 
-To associate the `<label>` with an `<input>` element, you need to give the `<input>` an `id` attribute. The `<label>` then needs a `for` attribute whose value is the same as the input's `id`.
+To explicitly associate a `<label>` element with an `<input>` element, you first need to add the `id` attribute to the `<input>` element. Next, you add the `for` attribute to the `<label>` element, where the value of `for' is the same as the `id` in the `<input>` element.
```
Comment (hamishwillee) [inline files/en-us/web/html/element/label/index.md:25]:
```suggestion
To explicitly associate a `<label>` element with an `<input>` element, you first need to add the `id` attribute to the `<input>` element. Next, you add the `for` attribute to the `<label>` element, where the value of `for` is the same as the `id` in the `<input>` element.
```

File: files/en-us/web/html/element/label/index.md [hunk 6]
```diff
@@ -17,12 +17,12 @@ The **`<label>`** [HTML](/en-US/docs/Web/HTML) element represents a caption for
 
 {{EmbedInteractiveExample("pages/tabbed/label.html", "tabbed-shorter")}}
 
-Associating a `<label>` with an {{htmlelement("input")}} element offers some major advantages:
+Associating a `<label>` with a form control, such as {{htmlelement("input")}} or  {{htmlelement("textarea")}}   offers some major advantages:
```
Comment (hamishwillee) [inline files/en-us/web/html/element/label/index.md:20]: Remove a couple of spaces
```suggestion
Associating a `<label>` with a form control, such as {{htmlelement("input")}} or {{htmlelement("textarea")}} offers some major advantages:
```

**Review by hamishwillee — APPROVED** [review]
LGTM - thanks. @dipikabh it looks to me like all your requests have been addressed so merging.

**Pull request merged**